README.md

```diff
@@ -21,7 +21,7 @@ It eliminates the need to copy, archive or move files.
 
 ## Current status
 
-Currently the plugin is under development. See the links to get more info about the current status.
+Version 1.0.0 has been released!
```

Comment:
I have seen some projects make use of a version badge. I have no idea how to do that myself.